instrumentation/gwt-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/gwt/GwtInstrumentationModule.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.gwt;
+
+import static io.opentelemetry.javaagent.instrumentation.gwt.GwtTracer.tracer;
+import static io.opentelemetry.javaagent.tooling.bytebuddy.matcher.ClassLoaderMatcher.hasClassesNamed;
+import static java.util.Collections.singletonList;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
+import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.javaagent.tooling.InstrumentationModule;
+import io.opentelemetry.javaagent.tooling.TypeInstrumentation;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+@AutoService(InstrumentationModule.class)
+public class GwtInstrumentationModule extends InstrumentationModule {
+
+  public GwtInstrumentationModule() {
+    super("gwt", "gwt-2.0");
+  }
+
+  @Override
+  public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
+    // class added in gwt 2.0
+    return hasClassesNamed("com.google.gwt.uibinder.client.UiBinder");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return singletonList(new RpcInstrumentation());
+  }
+
+  public static class RpcInstrumentation implements TypeInstrumentation {
+
+    @Override
+    public ElementMatcher<? super TypeDescription> typeMatcher() {
+      return named("com.google.gwt.user.server.rpc.RPC");
+    }
+
+    @Override
+    public Map<? extends ElementMatcher<? super MethodDescription>, String> transformers() {
+      Map<ElementMatcher<MethodDescription>, String> transformers = new HashMap<>();
+
+      transformers.put(
+          named("invokeAndEncodeResponse")
+              .and(takesArguments(5))
+              .and(takesArgument(0, named(Object.class.getName())))
+              .and(takesArgument(1, named(Method.class.getName())))
+              .and(takesArgument(3, named("com.google.gwt.user.server.rpc.SerializationPolicy")))
+              .and(takesArgument(4, named(int.class.getName()))),
```

Comment:
There's no need for `named()` matcher when referencing standard Java types:
```suggestion
              .and(takesArgument(0, Object.class))
              .and(takesArgument(1, Method.class))
              .and(takesArgument(3, named("com.google.gwt.user.server.rpc.SerializationPolicy")))
              .and(takesArgument(4, int.class)),
```

Reply:
Thanks, didn't know that. Should be fixed now.